include/openenclave/internal/sgxsign.h

```diff
@@ -35,6 +35,32 @@ oe_result_t oe_sgx_sign_enclave(
     size_t pem_size,
     sgx_sigstruct_t* sigstruct);
 
+/**
+ * Digitally sign the enclave with the given hash using an openssl engine
+ *
+ * This function digitally signs the enclave whose hash is given by the
+ * **mrenclave** parameter. The signing key is given by the **engine_id** and
+ * related **key_id** parameter. If successful, the function writes the
+ * signature into the
+ * **sigstruct** parameter (an SGX signature structure).
+ *
+ * @param mrenclave[in] hash of the enclave being signed
+ * @param engine_id[in] text name of the engine to use
+ * @param key_id[in]    integer handle for the key to use.
+ * @param sigstruct[out] the SGX signature
```

Comment:
nit: inconsistent spacing.  Either put one space after the ] in 49 or else make all the descriptions line up where line 50 doesn't currently.

Reply:
done.
At one point the text lined up but scripts/format-code "fixed" it. The only safe option seems to be one space, though I don't think that is very readable (IMO). But your point is taken that it needs to be one or the other.